.gitignore

```
@@ -5,3 +5,4 @@ _site
 vendor
 Gemfile.lock
 .*.sw?
+_docs/~$atiscloud.md
```

Make this generic; `.gitignore` allows for [shell-style wildcard matching](https://git-scm.com/docs/gitignore).
```suggestion
_docs/~$*.md
```